Source/WebCore/page/MemoryRelease.cpp

```diff
@@ -88,6 +88,10 @@ static void releaseNoncriticalMemory(MaintainMemoryCache maintainMemoryCache)
 
     InlineStyleSheetOwner::clearCache();
     HTMLNameCache::clear();
+
+#if PLATFORM(WPE)
+     GCController::singleton().garbageCollectSoon();
```

Comment:
Why this? The problematic situation is being handled with the MemoryPressureMonitor sending critical memory pressure events, which is handled in releaseCriticalMemory and not here.

Reply:
This was a change from 2.28. I have re-tested without it (but with all other changes, including the ones not part of this pull request anymore) and memory remains within limits. I'll update the pull request to remove this particular change.

Reply:
Done